dspace/config/spring/api/orcid-authority-services.xml

```diff
@@ -27,6 +27,10 @@
     <alias name="org.dspace.authority.AuthoritySearchService" alias="org.dspace.authority.indexer.AuthorityIndexingService"/>
 
     <bean id="dspace.DSpaceAuthorityIndexer" class="org.dspace.authority.indexer.DSpaceAuthorityIndexer"/>
+    <bean id="FunderAuthorityIndexerInterface" class="org.dspace.authority.indexer.FunderAuthorityIndexerInterface"/>
```

Comment:
Please set the scope of these spring beans (including DSpaceAuthorityIndexer) to "prototype" to prevent concurrency issues.